http/oidc/src/main/java/org/wildfly/security/http/oidc/LogoutHandler.java

```diff
@@ -63,19 +63,24 @@ protected boolean removeEldestEntry(Map.Entry<String, OidcClientConfiguration> e
     });
 
     boolean tryLogout(OidcHttpFacade facade) {
+        log.trace("tryLogout entered");
```

Comment:
Just to check, I know some of these trace and debug statements were added for debugging. Do we still want to keep all of these?

Reply:
I can remove them but I think they could be very helpful debugging customer issues with logout.

Reply:
Keeping ones that make sense is fine. Might be good to revisit the messages and make sure they have sufficient details/are consistent.

Reply:
done